packages/deployment/app/cloudformation.template.yml

```diff
@@ -1067,26 +1067,26 @@ Resources:
       Metrics:
         - Id: e1
           Expression: "FILL(m1, 0)"
-          Label: NormalizedActivityScheduleTime
+          Label: Normalized{{this.scaleIn.metricName}}
         - Id: m1
           MetricStat:
             Metric:
-              Namespace: AWS/States
-              MetricName: ActivityScheduleTime
+              MetricName: {{this.scaleIn.metricName}}
+              Namespace: {{# if this.scaleIn.namespace}}{{this.scaleIn.namespace}}{{ else }}AWS/States{{/if}}
               Dimensions:
                 - Name: ActivityArn
```

Comment:
This dimension is only going to match if they are scaling off of an activity-related metric. If they are triggering off of some other metric (size of an SQS queue, for example), then this "ActivityArn = ..." dimension is not going to match anything.

Reply:
The ActivityArn is the arn of the ECS task and is not related to the metric used.
e.g. ActivityArn: arn:aws:states:us-east-1:596205514787:activity:jl-cumulus-EcsTaskHelloWorldKesCma-Activity